sdk/react/panels/cast/CastConnectedScreen.js

```diff
@@ -0,0 +1,454 @@
+import PropTypes from 'prop-types';
+
+import React from 'react';
+import {
+  Animated, Text, TouchableOpacity, View, Image,
+} from 'react-native';
+import { BUTTON_NAMES, UI_SIZES, VALUES } from '../../constants';
+
+const Utils = require('../../utils');
+const styles = Utils.getStyles(require('../style/CastConnectedStyles.json'));
+const CastPlayPauseButtons = require('../../widgets/CastPlayPauseButtons');
+const ResponsiveDesignManager = require('../../responsiveDesignManager');
+
+const BottomOverlay = require('../../bottomOverlay');
+
+class CastConnectedScreen extends React.Component {
+  static propTypes = {
+    playhead: PropTypes.number.isRequired,
+    duration: PropTypes.number.isRequired,
+    live: PropTypes.bool.isRequired,
+    width: PropTypes.number.isRequired,
+    height: PropTypes.number.isRequired,
+    volume: PropTypes.number.isRequired,
+    fullscreen: PropTypes.bool.isRequired,
+    cuePoints: PropTypes.arrayOf(PropTypes.double).isRequired,
+    stereoSupported: PropTypes.bool.isRequired,
+    multiAudioEnabled: PropTypes.bool.isRequired,
+    playbackSpeedEnabled: PropTypes.bool.isRequired,
+    selectedPlaybackSpeedRate: PropTypes.string.isRequired,
+    handlers: PropTypes.shape({
+      onSwitch: PropTypes.func,
+      onPress: PropTypes.func,
+      onScrub: PropTypes.func,
+      handleVideoTouchStart: PropTypes.func,
+      handleVideoTouchMove: PropTypes.func,
+      handleVideoTouchEnd: PropTypes.func,
+      handleControlsTouch: PropTypes.func,
+      handleShowControls: PropTypes.func,
+    }).isRequired,
+    screenReaderEnabled: PropTypes.bool.isRequired,
+    availableClosedCaptionsLanguages: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+    config: PropTypes.object.isRequired,
+    localizableStrings: PropTypes.object.isRequired,
+    locale: PropTypes.string.isRequired,
+    playing: PropTypes.bool.isRequired,
+    loading: PropTypes.bool.isRequired,
+    initialPlay: PropTypes.bool.isRequired,
+    onDisconnect: PropTypes.func.isRequired,
+    deviceName: PropTypes.string.isRequired,
+    inCastMode: PropTypes.bool.isRequired,
+    previewUrl: PropTypes.string.isRequired,
+  };
+
+  componentWillMount() {
+    const { height } = this.props;
+
+    this.state = {
```

Comment:
I think it’s not a good practice to set state without `setState`

Reply:
the same as above

Reply:
Delete it at all because of we don't use animation in this screen.